### PR TITLE
Improving restart from parameters file test

### PR DESCRIPTION
**Pull request by jbrillon**
PR to improve test `MPI_VISCOUS_TAYLOR_GREEN_VORTEX_RESTART_FROM_PARAMETER_FILE_CHECK`, previously required manually copying a restart `.prm` file generated from `MPI_VISCOUS_TAYLOR_GREEN_VORTEX_RESTART_CHECK` and manually changing the `test_type`.  This PR eliminates the need to do that process and removes a huge `.prm` file from the `tests` directory. Closes issue #130.

### CODE REVIEW

**Review by PranshulThakur — APPROVED** [review]
LGTM! It was a nice idea to execute the shell script while adding the test.

File: tests/integration_tests_control_files/taylor_green_vortex_integration/CMakeLists.txt [hunk 1]
```diff
@@ -28,10 +28,16 @@ add_test(
...
 )
 # ----------------------------------------
-configure_file(viscous_taylor_green_vortex_restart_from_parameter_file_check.prm viscous_taylor_green_vortex_restart_from_parameter_file_check.prm COPYONLY)
+configure_file(viscous_taylor_green_vortex_restart_from_parameter_file_check.sh
+viscous_taylor_green_vortex_restart_from_parameter_file_check.sh COPYONLY)
 add_test(
   NAME MPI_VISCOUS_TAYLOR_GREEN_VORTEX_RESTART_FROM_PARAMETER_FILE_CHECK
-  COMMAND mpirun -np ${MPIMAX} ${EXECUTABLE_OUTPUT_PATH}/PHiLiP_3D -i ${CMAKE_CURRENT_BINARY_DIR}/viscous_taylor_green_vortex_restart_from_parameter_file_check.prm
+  COMMAND bash -c 
+  "./viscous_taylor_green_vortex_restart_from_parameter_file_check.sh ${EXECUTABLE_OUTPUT_PATH}
```
Comment (PranshulThakur) [inline tests/integration_tests_control_files/taylor_green_vortex_integration/CMakeLists.txt:36]: Just got a thought. Does this trick work while running ctest on narval?

Comment (jbrillon) [reply tests/integration_tests_control_files/taylor_green_vortex_integration/CMakeLists.txt:36]: Yes, it's done the same way as: https://github.com/dougshidong/PHiLiP/blob/0056e05976a85ba4ff9d623cac81d4be6c3719f6/tests/integration_tests_control_files/reduced_order/CMakeLists.txt#L9-L19

**Review by donovan97 — APPROVED** [review]
LGTM. Nice use of Bash.

**Pull request merged**